### PR TITLE
STG79 Preview

### PR DESCRIPTION
**Pull request by tasherif-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py [hunk 1]
```diff
@@ -175,6 +175,7 @@ class ListContainersIncludeType(with_metaclass(_CaseInsensitiveEnumMeta, str, En
 
     METADATA = "metadata"
     DELETED = "deleted"
+    SYSTEM = "system"
```
Comment (seanmcc-msft) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py:178]: Is this supposed to be here?  This is an STG 80 feature.  If this class isn't public it doesn't matter.

**Review by seanmcc-msft — APPROVED** [review]

File: sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py (hunk 1 above)
Comment (tasherif-msft) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py:178]: It is not public

**Review by seanmcc-msft — APPROVED** [review]

**Pull request merged**